src/watts/plugin.py

```diff
@@ -79,6 +79,9 @@ def __call__(self, params: Parameters = None, name: str = '', verbose=True, **kw
             Name associated with execution of plugin
         verbose
             Whether to print execution information
+        cleanup
+            Determines whether the temporary directory will be cleaned up after
+            execution.
```

Comment:
Maybe change to: Determines whether the temporary directory will be cleaned up **by watts immediately** after execution